html_css/intermediate_css/enhancing-design-with-css.md

```diff
@@ -18,7 +18,7 @@ One more note, the **Formal Syntax** section here is _crazy_. Don't let it deter
 #### Borders
 At this point, you've probably already encountered `border` and `border-radius`. The `border` property is another shorthand, but it is _much_ less complicated than the background shorthand. For borders, basically you just need to define a size, style and color.
 
-`border-radius` is the property that is used to create rounded corners on things. As you'll see in the docs, it's possible to get fancy and define different radii for each corner of an element, but this is rarely useful. Store that information in the category of "things I'll look up if I ever need it".
+`border-radius` is the property that is used to create rounded corners on things. As you'll see in the docs, it's possible to get fancy and define a different radius for each corner of an element, but this is rarely useful. Store that information in the category of "things I'll look up if I ever need it".
```

Comment:
I believe "radii" is used  as the [plural of radius](https://www.merriam-webster.com/dictionary/radii) here, so it's usage is correct.

Although, "different radii" and "a different radius" both work here. Do you have a preference of which phrasing should be used?

Reply:
@01zulfi You are correct, this can be closed.